docs/_source/tutorials/notebooks/labelling-textclassification-sentencetransformers-weaksupervision.ipynb

```diff
@@ -203,10 +210,11 @@
     "\n",
     "5. Once you are satisfied with your extended weak labels, use the extended matrix of the `WeakLabels` instance with your library/method of choice to build a training set or even train a downstream text classification model. You can iterate between this step and step 4 to try several thresholds and embeddings possibilities until you achieve a satisfactory result.\n",
     "\n",
-    "This guide shows you an end-to-end example using Snorkel. You could alternatively use any other label model available in Argilla. If you are interested in learning about other options, please check our [weak supervision guide](../../guides/techniques/weak_supervision.html)."
+    "This guide shows you an end-to-end example using Snorkel. You could alternatively use any other label model available in Argilla. If you are interested in learning about other options, please check our [weak supervision guide](https://docs.argilla.io/en/latest/guides/programmatic_labeling_with_rules.html)."
```

Comment:
Hi @garimau, we should make this relative URLs to the local files in the website. Something like this `/guides/programmatic_labeling_with_rules` should work. Could you check this with the `sphinx-autobuild` thingy?